### ISSUE TITLE
Where is the I2V training code?

### ISSUE THREAD

**Issue by jackieyung** [baaivision/NOVA#33]
Thanks for your excellent work. 

I want to train an I2V model based on your method, but I only find the T2V training code in https://github.com/baaivision/NOVA/blob/main/docs/training.md. Could you please release the I2V training code?

Thanks a lot.

**Comment by PhyscalX** [comment]
Hi, @jackieyung 

In NOVA framework, T2V and I2V tasks share the same training pipeline, i.e., T2V=T2I+I2V.
To discard T2I loss, remove it from the loss dict or multiply it by zero. [[HERE]](https://github.com/baaivision/NOVA/blob/2cd254afdedae8670ccb492c28c53caa08f7b5a0/diffnext/models/transformers/transformer_3d.py#L99).
